Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -169,5 +170,20 @@ private record Request(CancellationToken CancellationToken) {
     private record LoadRequest(TextDocumentItem TextDocument, CancellationToken CancellationToken) : Request(CancellationToken);
 
     private record VerifyRequest(DafnyDocument OriginalDocument, CancellationToken CancellationToken) : Request(CancellationToken);
+
+    private class VerificationProgressReporter : IVerificationProgressReporter {
+      public ICompilationStatusNotificationPublisher publisher { get; init; }
```

Comment:
Either make these properties `readonly private`, or capitalize them `Publisher`, `Document`
This is the convention that is being used in similar files.